src/docfx/Errors.cs

```diff
@@ -43,6 +43,12 @@ public static Error RedirectionUrlInvalid(SourceInfo<string> source)
         public static Error RedirectionUrlConflict(SourceInfo<string> source)
             => new Error(ErrorLevel.Warning, "redirection-url-conflict", $"The '{source}' appears twice or more in the redirection mappings", source);
 
+        /// <summary>
+        /// The dest to redirection url does not match any files's publish URL, but the redirect_with_id flag has been set as true
+        /// </summary>
+        public static Error RedirectionUrlNotExisted(SourceInfo<string> source)
+            => new Error(ErrorLevel.Error, "redirection-url-not-existed", $"The redirect url '{source}' does not match any file's publish URL, but the redirect_with_id flag has been set as true", source);
```

Comment:
it says report `warning` in https://dev.azure.com/ceapex/Engineering/_workitems/edit/112321, upgraded to `error`?

Reply:
Yes, since it should break the build of this file, so upgraded to `error`